src/controller/apply.ts

```
@@ -0,0 +1,20 @@
+import express, { Response } from 'express';
+import apply from '../service/apply';
+import { verifyJWT } from '../middleware/jwt';
+import { AuthenticatedRequest } from '../types';
+import { apiLimit, getApiLimit } from '../middleware/limit';
+import { ApplyLoanRequest } from '../types/apply';
+
+const app = express.Router();
+
+app.post('/', apiLimit, verifyJWT, (req: AuthenticatedRequest<{}, {}, ApplyLoanRequest>, res: Response) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)